src/features/post/components/CommentCard.js

```diff
@@ -0,0 +1,49 @@
+import "../styles.css";
+import { Avatar } from "../../../Components/Avatar/Avatar";
+import { useNavigate } from "react-router-dom";
+import { useSelector } from "react-redux";
+const CommentCard = ({ comment, postId }) => {
+  const users = useSelector((state) => state.user.users);
+  const currentUser = users?.find(
+    (dbUser) => dbUser.username === comment.username
```

Comment:
The username has a chance of not being unique, maybe you should do this check on the user ID. Please review.